examples/asr/asr_adapters/README.md

```diff
@@ -58,3 +58,51 @@ graph TD
 ```
 
 **Note**: If you with to evaluate the base model (with all adapters disabled), simply pass `model.adapter.adapter_name=null` to the config of this script to disable all adapters and evaluate just the base model.
+
+# Scoring and Analysis of Results
```

Comment:
This basic usage documentation should be inside of the scoring and analysis.py script probably.

Reply:
This is done.

For now, the basic usage documentation is present in the README as well as inside the script. Should I remove it from the README?